Added more parts for SD Card Logger schematic

mkazc opened the pull request:
Added schematic for Linear Voltage Regulator, 5V-3V3 Level Shifter, started on the SD card socket. Connections to ATMEGA, added debugging LEDs

mkazc committed:
Added schematic for Linear Voltage Regulator, 5V-3V3 Level Shifter, started on the SD card socket. Connections to ATMEGA, added debugging LEDs

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
ATMEGA LEDs, SD Pro/Detect

jack-greenberg reviewed (changes requested):
Nice work! A few comments:

* Use no-connects on your ATMega16m1
* Be sure to run annotation so that you have all of your components labeled (like R1 or U3, instead of a bunch of R? and U?)
* Should all the capacitors near pin 4 be places near pin 4? or just one of them?

These are the first few comments. Make the changes, do another commit, and push again (`git push origin main`), and then re-request my review.

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
New Level Shifter, Changes from Feedback

mkazc commented:
![Screenshot from 2020-11-30 21-04-57](https://user-images.githubusercontent.com/29257011/100688213-d2fa8b00-334f-11eb-951d-1a8b5bb8d397.png)
Screenshot of Schematic

mkazc committed:
Fixed Annotations

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
Design Checklist, Edit Resistors

mkazc committed:
Removed and replaced rescue

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
Completed PCB Design Checklist, waiting on connector

mkazc committed:
Began PCB layout, and schematic layout boxing

mkazc commented:
[canlogger2-8-2021.pdf](https://github.com/olin-electric-motorsports/AdvancedResearch/files/5961781/canlogger2-8-2021.pdf)

mkazc committed:
Small changes: 3.3V->3V3, added test points to Debug LEDS, etc.

Following recommendations given by Wesley, pg. 2 of this file:
https://drive.google.com/file/d/12oTchTOrF_66K-js-7AgllKoxS_4IslO/view.

mkazc committed:
PCB Layout: grouped together functional blocks, added GND to connector

Grouped functional blocks together by component in PCB layout, connected connector to ground. Layout is currently off board.

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
Worked PCB Layout, small schematic changes

+Layout: Everything on board, placed some wires and vias
+Schematic: Moved locations of P_LEDs on ATMEGA

mkazc committed:
Merge branch 'main' of https://github.com/olin-electric-motorsports/AdvancedResearch into main

mkazc committed:
Completed initial layout, wire connections

mkazc committed:
Fixed via wire for SD_Protect, edited labels

mkazc committed:
Rearranging layout, changed 3.3V to 3V3

mkazc committed:
Completed 2nd revision of PCB layout
